charts/polkadot-watcher/templates/alertrules.yaml

```diff
@@ -23,7 +23,7 @@ spec:
         message: 'Blocks were not produced for 6:11 hours by {{`{{ $labels.name }}`}}'
```

Comment:
Same the message should be updated to match the new value.

Reply:
updated